qmri_mpm/sub-01/fmap/sub-01_phasediff.json

```diff
@@ -0,0 +1,25 @@
+{
+  "Manufacturer": "SIEMENS ",
+  "ManufacturersModelName": "Prisma_fit",
+  "DeviceSerialNumber": "167025",
+  "StationName": "MRC35437",
+  "MagneticFieldStrength": 3,
+  "ScanningSequence": "GR",
+  "SequenceName": "*fm2d2r ",
+  "PulseSequenceDetails": "gre_field_mapping_1acq_rl ",
+  "RepetitionTimeExcitation": 1.02,
+  "EchoTime1": 0.01246,
+  "EchoTime2": 0.01246,
```

Comment:
https://github.com/bids-standard/bids-examples/pull/245/checks?check_run_id=1872898390

This seems like an error unrelated to missing qMRI coverage

![image](https://user-images.githubusercontent.com/9084751/107541061-087ce980-6bc7-11eb-9909-f78e06caeed3.png)